package.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
I added that to push it to Heroku for testing, but I didn't mean to add it to the commits, my bad!